tests/test-verify-volume-mount.yml

```diff
@@ -30,6 +30,30 @@
       _storage_test_volume_present and
       storage_test_volume.fs_type == 'swap' else 0 }}"
 
+- name: Get information about the mountpoint directory owner
```

Comment:
I suggest using the `stat` module here https://docs.ansible.com/ansible/latest/collections/ansible/builtin/stat_module.html#ansible-collections-ansible-builtin-stat-module
```yaml
- name: Get information about the mountpoint directory
  stat:
    path: "{{ storage_test_volume.mount_point }}"
  register: storage_test_found_mount_stat
  when: _storage_test_volume_present and
        storage_test_volume.mount_point and
        (storage_test_volume.mount_user or storage_test_volume.mount_group or
         storage_test_volume.mount_permissions)
```

Reply:
Modified.